src/components/home/elements/SplitToken.vue

```diff
@@ -56,6 +56,10 @@
                                      v-else-if="!isInteger()">
               Integer number is allowed.
             </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="isGreaterThanMax(this.newUnity)">
+              Unity is too big, and it should be no more than 2^63-1.
```

Comment:
change the words to `Unity is too large. Please change it to an integer less than 2^63.`